themes/wim/components/wim/_block--menu-top-tasks.scss

```diff
@@ -24,35 +24,40 @@
         position: static;
         cursor: pointer;
         margin-bottom: 0;
-
-        @include MQ($screen-sm) {
-          &.open-default,
-          &:hover {
+        
+        &.open,
+        &.open-default {
+          .dropdown-menu {
+            display: block;
+          }
+          
+          .dropdown-toggle {
             background-color: $brand-primary;
-            .nolink {
+            color: #fff;
+
+            &:focus,
+            &:active,
+            &:hover {
+              background-color: $brand-primary;
               color: #fff;
             }
-            .dropdown-menu {
-              display: block;
-            }
-          }
-          &.open-default,
-          &:hover,
-          &.open {
-            .nolink {
+            @include MQ($screen-sm) {
               &:before {
                 display: block !important;
               }
             }
           }
         }
-        .nolink {
+        .dropdown-toggle {
           padding: 18px 10px;
           color: #000;
-          font-family: "Roboto Slab", "Droid Sans", Arial, sans-serif;
+          font-family: "Roboto Slab", "Droid Sans", Arial, sans-serif; //@todo use variable
```

Comment:
@Crazylord We also want te font to be easily changed site wide. So also the right variables for font settings, so we only have to change some variables in future themes.